crates/bevy_gltf/src/loader.rs

```diff
@@ -550,6 +553,42 @@ fn load_node(
             }
         }
 
+        if let Some(light) = gltf_node.light() {
+            match light.kind() {
+                gltf::khr_lights_punctual::Kind::Directional => {
+                    parent.spawn_bundle(DirectionalLightBundle {
+                        directional_light: DirectionalLight {
+                            color: Color::from(light.color()),
+                            // NOTE: KHR_punctual_lights defines the intensity units for directional
+                            // lights in lux (lm/m^2) which is what we need.
+                            illuminance: light.intensity(),
+                            ..Default::default()
+                        },
+                        ..Default::default()
+                    });
+                }
+                gltf::khr_lights_punctual::Kind::Point => {
+                    parent.spawn_bundle(PointLightBundle {
+                        point_light: PointLight {
+                            color: Color::from(light.color()),
+                            // NOTE: KHR_punctual_lights defines the intensity units for point lights in
+                            // candela (lm/sr) which is luminous intensity and we need luminous power.
+                            // For a point light, luminous power = 4 * pi * luminous intensity
+                            intensity: light.intensity() * std::f32::consts::PI * 4.0,
+                            range: light.range().unwrap_or(20.0),
+                            radius: light.range().unwrap_or(20.0),
```

Comment:
```suggestion
                            radius: light.range().unwrap_or(0.0),
```
I think it should be 0 to match the default for `PointLight`
https://github.com/bevyengine/bevy/blob/1d0d8a3397bd6fc2c14d42ffd0668d2443748912/crates/bevy_pbr/src/light.rs#L57